tests/models/idefics/test_modeling_idefics.py

```diff
@@ -316,7 +316,6 @@ def prepare_pixel_values(self):
         return floats_tensor([self.batch_size, self.num_channels, self.image_size, self.image_size])
 
     @require_torch_sdpa
-    @slow
```

Comment:
i think this test dont need skip anymore since we dont check if model has SDPA layers within this test anymore. But it can be skipped due to the same flakiness

Reply:
It still has input preparation issues, where I added below to another model test class

> "Idefics requires both text and image inputs which is currently not done in this test."

As mentioned in a reply to Joao's comment, let's try to do it in a follow up PR